src/Framework/README.md

```diff
@@ -0,0 +1,25 @@
+# Framework
+
+Contains packages that provide reference assembles, default sets APIs for building ASP.NET Core applications, and assets used for self-contained deployments.
+
+## Description
+
+The following contains a description of each sub-directory in the `DataProtection` directory.
+
+- `App.Ref.Internal`: Contains internal reference assemblies, documentation, and other design-time assets
```

Comment:
This package doesn't actually contain anything, it's sole purpose is to create a package identifying the non-stable version of AspNetCore at the time we last build the targeting pack

Reply:
Thanks updated, is that better?